src/test/resources/feature/subscription.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Subscription test
+
+  Scenario: Subscribe to 'Learn faster' newsletter
+    Given I navigate to the main page
+    And I accept the data handling dialog box
+    And I click on the Subscribe button
```

Comment:
The first three steps can go into Background I think.

Reply:
Fixed in 384a184a8511e7509c867e0c58f7570d33cd2e6c!